### DISCUSSION TITLE
真的烦了

### DISCUSSION THREAD

**Discussion by ajjdxns** [Meloong-Git/PCL#3623] [讨论]
我在写一个启动器，但是想要写一个外部验证，结果半天搞不出。来借鉴一下PCL的，可是我看不懂。怎么办好呢？

**Comment by allMagicNB** [comment]
你用啥语言写的？

**Reply by ajjdxns** [reply]
python

**Comment by zkitefly** [comment]
外部验证？外置登录吗？如果是，可以查看 authlib-injector 的 启动器技术规范

https://github.com/yushijinhun/authlib-injector/wiki/%E5%90%AF%E5%8A%A8%E5%99%A8%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83


> 概述
> 本文旨在为启动器实现 authlib-injector 规范提供技术指导。由于该功能需要调用 Yggdrasil API，因此建议您在阅读本文前，先阅读 [Yggdrasil 服务端技术规范](https://github.com/yushijinhun/authlib-injector/wiki/Yggdrasil-%E6%9C%8D%E5%8A%A1%E7%AB%AF%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83)。
> 
> 在启动器中，本登录方式可以被称为【外置登录（authlib-injector）】或【authlib-injector 登录】。我们推荐您使用含义更为明确的前者。

**Reply by Deep-Dark-Forest** [reply]
他说的可能是 littleskin 之类的登录方式

**Comment by allMagicNB** [comment]
https://github.com/Blessing-Studio/minecraft-launch-p

**Reply by hejiehao** [reply]
这个好像没有外置登录的

**Reply by allMagicNB** [reply]
> 这个好像没有外置登录的

额……

**Comment by wuliaodexiaoluo** [comment]
统一通行证开发文档：https://login.mc-user.com:233/index/doc
littleskin 使用的服务端开发文档：https://github.com/yushijinhun/authlib-injector/wiki/%E5%90%AF%E5%8A%A8%E5%99%A8%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83

**Comment by MoYuan-CN** [comment]
有需要的话过两天我开源一下我的正版登录 Python 模块
